azurerm/resource_arm_iothub_test.go

```diff
@@ -113,6 +113,30 @@ func TestAccAzureRMIotHub_customRoutes(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMIotHub_fallbackRoute(t *testing.T) {
+	resourceName := "azurerm_iothub.test"
+	rInt := tf.AccRandTimeInt()
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMIotHubDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMIotHub_fallbackRoute(rInt, testLocation()),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMIotHubExists(resourceName),
+				),
```

Comment:
Could we check some of the values returned in state?? ie:
```go
resource.TestCheckResourceAttr("azurerm_iothub.test", "fallback_route.0.name", "fallback"),
...
```

and such for the rest of them?

Reply:
Added additional checks for all the fallback fields.